lib/ui/pedals/clouds.lua

```diff
@@ -0,0 +1,98 @@
+--- CloudsPedal
+-- @classmod CloudsPedal
+
+local ControlSpec = require "controlspec"
+local UI = require "ui"
+local Pedal = include("lib/ui/pedals/pedal")
+local Controlspecs = include("lib/ui/util/controlspecs")
+
+local CloudsPedal = Pedal:new()
+-- Must match this pedal's .sc file's *id
+CloudsPedal.id = "clouds"
+
+function CloudsPedal:new(bypass_by_default)
+  local i = Pedal:new(bypass_by_default)
+  setmetatable(i, self)
+  self.__index = self
+
+  i.sections = {
+    {"Pitch/Pos", "Size/Dens"}, 
+    {"Tex/Spread", "Mode/Feedback"},
+    i:_default_section(),
+  }
+  i:_complete_initialization()
+
+  return i
+end
+
+function CloudsPedal:name(short)
+  return short and "CLDS" or "Clouds"
+end
+
+function CloudsPedal.params()
+  local id_prefix = CloudsPedal.id
+
+  local pitch_control = {
+    id = id_prefix .. "_pit",
+    name = "Pitch",
+    type = "control",
+    controlspec = ControlSpec.new(-48, 48, "lin", 1, 0, ""),
```

Comment:
unit "st" is standard for semitones throughout the app

Reply:
FWIW - i'm not sure the value is actually semitones. Manual does not say and I'm not sure where to dig for that in the source